### PR TITLE
RHDP-426 Send Webhook after the tests finished in OpenShift CI

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

File: .ci/oci-e2e-deployment.sh [hunk 1]
```diff
@@ -82,7 +82,25 @@ function checkHASGithubOrg() {
...
+function prepareWebhookVariables() {
+    # Download config file
+    wget https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/main/webhookConfig.yml
+    #Export variables
+    export webhook_salt=123456789
+    export webhook_target=https://smee.io/JgVqn2oYFPY1CF
+    export webhook_repositoryURL=https://github.com/redhat-appstudio-qe/infra-deployments
+    export webhook_repositoryFullName=redhat-appstudio-qe/infra-deployments
+    # Delete temp file
+    rm -f temp.yml
```
Comment (Michkov) [inline .ci/oci-e2e-deployment.sh:97]: `curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/main/webhookConfig.yml | envsubst > webhookConfig.yml`

Comment (jkopriva) [reply .ci/oci-e2e-deployment.sh]: Thank you @Michkov ! Good hint :)

**Pull request merged**